### PR TITLE
fix: AR Senate Committee list filtering

### PR DESCRIPTION
**Pull request by elseagle**

### CODE REVIEW

File: scrapers_next/ar/committees.py [hunk 1]
```diff
@@ -149,19 +149,32 @@ def process_item(self, item):
...
     source = URL("https://senate.arkansas.gov/senators/committees/")
-    selector = CSS("ins > ul > li", num_items=45)
+    selector = CSS("div > ul > li", num_items=54)
+    committee_text = None
 
     def process_item(self, item):
         comm_name = CSS("a").match(item)[0].text_content().strip()
 
+        # we skip the non senate committees from the first time we encounter them, which means senate list is done
+        if (
```
Comment (johnseekins) [inline scrapers_next/ar/committees.py:159]: `if self.committee_text in ["Task Forces", "Joint Committees"]`?

File: scrapers_next/ar/committees.py [hunk 2]
```diff
@@ -149,19 +149,32 @@ def process_item(self, item):
...
+        ):
+            self.skip()
+
         previous_sibs = item.getparent().itersiblings(preceding=True)
         for sib in previous_sibs:
-            if len(sib.getchildren()) == 0:
+            # h4 is the tag of the committee type
+            if sib.tag == "h4":
                 chamber_type = sib.text_content().strip()
+                if chamber_type == "":
```
Comment (johnseekins) [inline scrapers_next/ar/committees.py:170]: `if not chamber_type:`?

Comment (elseagle) [reply scrapers_next/ar/committees.py:167]: Makes sense, my thought was zeroed in on "" because that was the actual value causing issues - switching it

Comment (johnseekins) [reply scrapers_next/ar/committees.py:167]: Well...`if not` just checks all `False`-y values. If _only_ `""` is appropriate, then I'm wrong.

**Review by johnseekins — APPROVED** [review]

**Review by NewAgeAirbender — APPROVED** [review]

**Pull request merged**